msm/src/serialization/interpreter.rs

```diff
@@ -83,23 +89,23 @@ pub trait InterpreterEnv<F: PrimeField> {
 /// ```
 /// And we can ignore the last 10 bits (i.e. `limbs2[78..87]`) as a field element
 /// is 254bits long.
-pub fn deserialize_field_element<F: PrimeField, Env: InterpreterEnv<F>>(
+pub fn deserialize_field_element<F: PrimeField, Ff: PrimeField, Env: InterpreterEnv<F, Ff>>(
     env: &mut Env,
-    limbs: [u128; 3],
+    limbs: [BigUint; 3],
```

Comment:
Any reason to change?

Reply:
The problem is that there is no `From` converting a field element to `u128` which I needed in several places. Generally it seems `BigUint` is not adding any significant overhead over `u128`, but it has significantly better interoperability with `Field` because we have `FieldHelpers` etc in `utils`.